apps/happy-blocks/src/pricing-plans/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { Edit } from './edit';
+import { Save } from './save';
+
+const blockAttributes = {};
+
+export function registerBlock() {
+	registerBlockType( 'happy-blocks/pricing-plans', {
+		title: __( 'Pricing plans', 'happy-blocks' ),
+		icon: 'leftright',
```

Comment:
Maybe the [`cart` icon](https://developer.wordpress.org/resource/dashicons/#cart) would make sense here?